data/json/construction.json

```
@@ -73,6 +73,56 @@
     "dark_craftable": true,
     "post_special": "done_trunk_plank"
   },
+  {
+    "type": "construction",
+    "id": "constr_paddy_field",
+    "group": "plow_a_paddy_field",
+    "category": "FARM_WOOD",
+    "required_skills": [ [ "survival", 1 ] ],
+    "time": "20 m",
+    "qualities": [ [ { "id": "DIG", "level": 1 } ] ],
+    "components": [ [ [ "clay_lump", 8 ] ], [ [ "water", 4 ], [ "water_clean", 4 ] ], [ [ "material_sand", 1 ], [ "material_soil", 2 ] ] ],
```

1 soil is 5 liters, 1 sand is 5 ml. Suggestion here uses the same ratio that "Fill Shallow Water With Dirt" uses, which is still barely any but eh, doesn't have to be exact given that would require 2000 sand:
```suggestion
    "components": [ [ [ "clay_lump", 8 ] ], [ [ "water", 4 ], [ "water_clean", 4 ] ], [ [ "material_sand", 20 ], [ "material_soil", 2 ] ] ],
```

---

Could work then, soil's more reliable to get than sand so.

---

Rice paddies require a hard layer of solid clay to trap water molecules in the pond. Sand is too porous to contain water so you want as much compacted clay as possible. There's a lot of research here but that's the jist of how a paddy field is done.